quickannotator/api/v1/utils/shared_crud.py

```diff
@@ -58,44 +59,132 @@ def get_annotation_query(model, scale_factor: float=1.0) -> Query:
 
     return query
 
-def upsert_tiles(image_id: int, annotation_class_id: int, tile_ids: List[int], pred_status: TileStatus = None):
+def reset_all_PROCESSING_tiles(annotation_class_id: int):
+    stmt = sqlalchemy.update(models.Tile).where(
+        models.Tile.annotation_class_id == annotation_class_id
+    ).values(
```

Comment:
is this correct? i don't see a constraint for this to operate only on PROCESSING tiles? what about e.g., DONE? this seems like it would reset them as well?

Reply:
You're right, this is incorrect.

Just pushed a fix + tests.

Note that this method isn't used yet but will be used if/when we implement a solution for perpetually PROCESSING tiles